Feature accordion categories (#1623)
SAP/luigi: docs/navigation-parameters-reference.md

  - **wrongPath** (string): the path that the user tried navigating to.
  - **wasAnyPathFitted** (bool): it is true if Luigi managed to fit a valid path which means **wrongPath** was only partially wrong. Otherwise it is false.

### defaults.sideNavAccordionMode
- **type**: boolean

> - **description**: defines the behaviour of categories whether multiple categories can be collapsed at the same time or similar to an accordion. (Default: false)

Reviewer: ```suggestion
- **description**: defines the behaviour of categories. When set to `true`, multiple categories can be collapsed at the same time and the navigation is similar to an accordion. 
- **default**: `false` 
```

Author: ```suggestion
- **description**: overrides the default behaviour of categories whether multiple categories can be collapsed. When set to `true`, only one category is collapsed. The navigation is similar to an accordion; when the user clicks another category the previously collapsed category is closed and the new one is opened.
- **default**: `false`
```

    ```
- **since**: 1.4.0

### sideNavAccordionMode
- **type**: boolean

> - **description**: overrides the default behaviour of categories whether multiple categories can be collapsed at the same time or similar to an accordion. (Default: false)

Reviewer: ```suggestion
- **description**: overrides the default behaviour of categories. When set to `true`, multiple categories can be collapsed at the same time and the navigation is similar to an accordion. 
- **default**: `false` 
```

Author: Hi @alexandra-simeonova , its actually the opposite. When set to true, only one category is open. The default functionality (false or not defined) is that any amount of collapsible categories can be open. :)

Author: ```suggestion
- **description**: overrides the default behaviour of categories whether multiple categories can be collapsed. When set to `true`, only one category is collapsed. The navigation is similar to an accordion; when the user clicks another category the previously collapsed category is closed and the new one is opened.```
